t/ui/12-needle-edit.t

```diff
@@ -270,7 +271,11 @@ sub check_flash_for_saving_logpackages {
     $driver->find_element('#flash-messages .btn-close')->click();
 }
 
-
+# put oddly sized png in place for the needle inst-timezone-text to be able to verify it was correctly loaded
+my $src_dir = path('t/testresults/00099/00099963-opensuse-13.1-DVD-x86_64-Build0091-kde');
+my $src_needle = $src_dir->child('reboot_after_install-1-reboot_after_install-131M4-diff2.png');
+my $src_needle_sha = sha1_sum $src_needle->slurp;
+$src_needle->copy_to($dir->child('inst-timezone-text.png'));
```

Comment:
Does this do the action in a temp directory or pollute the working directory? How about mocking the hashing as we don't really need to test that hashing files works, right?

Reply:
It pollutes the working directory like the whole test already does. The test cleans up the directory it pollutes before it runs so potential leftovers are not a problem. The particular directory is also ignored by Git. So this is not a big deal. Changing this would mean changing the whole test. 

Note that this does not invoke the hashing function to test hashing (as no hashing is done by the code we test here). I only use it to compare whether the expected file has been created when the needle is saved (and not a different file due to a regression).